CHANGELOG.md

```diff
@@ -3,7 +3,33 @@ All notable changes to this project will be documented in this file.
+- Benchmark tests for read-write splitting plugin ([PR #317](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/317))
+- Read-write splitting and reader load balancing benchmark and performance test results ([PR #340](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/340) and [PR #316](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/316))
+- Remaining Hikari and read-write splitting tests added to refactored test suite and removed old test suite ([PR #364](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/364)) and ([PR #377](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/377))
+- New plugin allowing users to override the driver name ([PR #371](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/371))
+- Default list of plugins added to the parameter 'wrapperPlugins' ([PR #332](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/332))
+- 2-node test cluster configuration ([PR #345](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/345))
+- Tests to ensure connection properties from the JDBC URL don't override existing credentials when reusing the AWSWrapperDataSource object ([PR #337](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/337))
+- Test if AWS supported region url exists as reported in issue [Issue #344](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/344). See test in [PR #351](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/351)
+- Hibernate integration test ([PR #352](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/352))
+- Documentation for the new connection plugin DriverMetadataConnectionPlugin and auroraConnectionTracker ([PR #378](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/378) and ([PR #360](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/360/files)))
+
+### :bug: Fixed
+- Close opened connections pointing to a dead node ([PR #298](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/298))
+- Unwrap Savepoint objects when passing them in as parameters ([PR #331](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/331))
+- Add null checks to WrapperUtils.getConnectionFromSqlObject() ([PR #348](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/348))
+
+### :crab: Changed
+- Lock initialization of AuroraHostListProvider ([PR #347](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/347))
+- Optimize thread locks and cache for EFM plugin ([PR #365](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/365))
+- Modify setReadOnly(true) failure scenarios ([PR #356](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/356))
+- Update Hibernate sample code to reflect changes in the wrapper source code ([PR #368](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/368))
+- Docs now specify the wrapper driver does not support blue/green deployments ([PR #376](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/376))
```

Comment:
I would suggest we use the same changelog sentence from the AWS JDBC Driver for MySQL. In addition, could we link to the README.md section containing this change, instead of the PR? I think we should do this for all documentation changes as that anchors customers to that specific section.

```suggestion
- Updated README.md to reflect that Amazon RDS Blue/Green Deployments are not supported.
```